### PR TITLE
Upgrade to Django 2.2 and Wagtail 2.5.1

### PR DESCRIPTION
**Pull request by tomijarvi**
Requires:
- [x] https://github.com/City-of-Helsinki/wagtailtinymce/pull/1
- [x] https://github.com/City-of-Helsinki/django-social-widgets/pull/1
- [x] Python 3.6 is required
- [ ] Elasticsearch needs to upgrade to minimum 2.x, 5.x or 6.x ([refs](http://docs.wagtail.io/en/v2.5.1/topics/search/backends.html#elasticsearch-backend))
- [ ] Postgressql needs to upgrade to minimum 9.4 ([refs](https://docs.djangoproject.com/en/2.1/releases/2.1/#dropped-support-for-postgresql-9-3))

### CODE REVIEW

File: requirements.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
--e git+https://github.com/City-of-Helsinki/wagtail-svgmap.git@v0.1.2#egg=wagtail_svgmap
--e git+https://github.com/City-of-Helsinki/wagtailtinymce.git@extensibility#egg=wagtailtinymce==4.4.3.0
--e git+https://github.com/City-of-Helsinki/django-social-widgets@master#egg=django_social_widgets
+-e git+https://github.com/tomijarvi/django-social-widgets@fix-1.9-depr#egg=django_social_widgets
```
Comment (vikoivun) [inline requirements.in]: I suppose we'll change these after deployment on server

Comment (tomijarvi) [reply requirements.in]: I can do this fix as dependencies changes have now merged.

**Comment by vikoivun** [comment]
We need to install newer elasticsearch on the server still. Maybe just run it from docker.

**Comment by vikoivun** [comment]
Agh. We seem to have conflicts. Can you resolve them still?

**Comment by tomijarvi** [comment]
Conflicts are now resolved, but this requires testing still.

**Comment by vikoivun** [comment]
Ah ok. Next week I assume?

**Comment by tomijarvi** [comment]
Yes, this goes next Monday

**Comment by tomijarvi** [comment]
This is now generally black-box tested and I haven't found any issues so far. I have used this in my demo as well. I would say that this is ready for merge.

**Pull request merged**